about.css

```diff
@@ -0,0 +1,661 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+}
+
+.bigheader {
+  color: #fa3e23;
+}
+
+.salute {
+  color: #f74d1c;
+}
+
+@media only screen and (max-width: 768px) {
+  .social,
+  .desktop_seemore,
+  .logoLion,
+  .lelogo,
+  .chinisse {
+    display: none;
+  }
+
+  .middle_bar {
+    position: absolute;
+    height: 5px;
+    background-color: black;
+    width: 40%;
+    border: none;
+    top: 95vh;
+    margin-right: 5%;
+  }
+
+  .mobile_nav {
+    padding-bottom: 10%;
+  }
+
+  .open {
+    color: #525252;
+    font-size: 40px;
+    font-weight: bolder;
+    margin-left: 10px;
+  }
+
+  .close {
+    display: none;
+    color: white;
+    font-size: 20px;
+  }
+
+  .hamburger {
+    color: transparent;
+    padding: 0;
+    width: auto;
+    z-index: 100;
+    background: transparent;
+    cursor: pointer;
+    border: none;
+    margin-right: 10px;
+  }
+
+  .closingIcon {
+    position: fixed;
+    top: 1rem;
+    right: 1rem;
+    padding: 4px;
+    margin-top: 5%;
+  }
+
+  .hide_y_scroll {
+    overflow-y: hidden;
+  }
+
+  .link_item {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    color: white;
+  }
+
+  .menu_list {
+    box-sizing: border-box;
+    position: fixed;
+    display: none;
+    transform: translateY(-100%);
+    transition: transform 0.2s;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 99;
+    background: rgba(53, 55, 64, 0.863);
+    backdrop-filter: blur(5px);
+    color: white;
+    list-style: none;
+    padding-top: 4rem;
+    margin-top: 0;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  .menu_list li {
+    margin-bottom: 5%;
+  }
+
+  .showMenu {
+    transform: translateY(0);
+    display: block;
+  }
+
+  .paragraph {
+    border: 4px solid #ececec;
+    padding: 20px 10px;
+    color: #7c7c7c;
+    background-color: white;
+  }
+
+  .message {
+    color: #7c7c7c;
+    text-align: center;
+  }
+
+  .place {
+    text-decoration: underline;
+  }
+
+  .banner {
+    padding-bottom: 12%;
+    text-align: center;
+  }
+
+  .partners {
+    background-color: #424146;
+    color: white;
+    text-align: center;
+    padding-top: 3%;
+    padding-bottom: 3%;
+    padding-left: 10%;
+    padding-right: 10%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .partener_logos {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+  }
+
+  .partener_logos li {
+    width: 30%;
+  }
+
+  header {
+    background-image: url("./pictures/main_big.png");
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    padding: 20px;
+  }
+
+  .programs {
+    background-color: white;
+    padding: 20px;
+    padding-bottom: 5%;
+  }
+
+  .red_bar {
+    border: 1px solid #9c3e36;
```

Comment:
You should use the exact same colors, for text, backgrounds, borders, etc, from the original design. Kindly fix the color of hr elements to #ec5242
<img width="432" alt="Screenshot 2023-03-14 at 10 00 12 PM" src="https://user-images.githubusercontent.com/25887253/225109965-f745faa2-1bcc-4f1b-9fca-da510436315d.png">
<img width="432" alt="Screenshot 2023-03-14 at 10 01 35 PM" src="https://user-images.githubusercontent.com/25887253/225110129-95594e94-dfa5-475d-9835-34f29aeab843.png">